### PR TITLE
Use Emotion

### PR DESCRIPTION
**Pull request by moonkii**
Install [Emotion](https://emotion.sh/) to use CSS-in-JS for React.

Move styles of `Header` component from CSS file to JSX file.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -46,5 +47,6 @@ module.exports = {
     'arrow-spacing': ['error', { before: true, after: true }],
     'react/prop-types': 'off',
     'jsx-a11y/label-has-associated-control': ['error', { assert: 'either' }],
+    'no-unused-vars': ['error', { varsIgnorePattern: 'jsx' }],
```
Comment (ahastudio) [inline .eslintrc.js]: for `@emotion/core`

File: src/App.test.jsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+jest.mock('./assets');
```
Comment (ahastudio) [inline src/App.test.jsx]: Jest Manual Mocks https://jestjs.io/docs/en/manual-mocks

File: src/assets/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import LogoImage from './images/logo.png';
+
+export {
+  LogoImage,
+
+  // TODO: When an asset is appended, remove default.
+  LogoImage as default,
```
Comment (ahastudio) [inline src/assets/index.js]: ESLint Rule - `import/prefer-default-export`
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

File: src/assets/__mocks__/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const LogoImage = jest.fn();
+
+// TODO: When an asset is appended, remove this.
+export const removeThis = jest.fn();
```
Comment (ahastudio) [inline src/assets/__mocks__/index.js]: ESLint Rule - `import/prefer-default-export`
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

**Comment by ahastudio** [comment]
Read these:
- [통합 스타일링 언어](https://blog.rhostem.com/posts/2017-06-24-unified-styling-language)
- [CSS-in-JS에 관해 알아야 할 모든 것](https://d0gf00t.tistory.com/22)

File: src/styles/utils.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export const mq = (width) => `@media (min-width: ${width}px)`;
```
Comment (ahastudio) [inline src/styles/utils.js]: https://github.com/emotion-js/emotion/blob/master/docs/media-queries.mdx

**Comment by moonkii** [comment]
Now CSS is with JS!
CSS-in-JS

**Pull request merged**